srtcore/core.cpp

```diff
@@ -6965,7 +6968,7 @@ int CUDT::receiveMessage(char* data, int len, SRT_MSGCTRL& w_mctrl, int by_excep
     {
         steady_clock::time_point tstime SRT_ATR_UNUSED;
         int32_t seqno;
-        if (stillConnected() && !timeout && (!m_pRcvBuffer->isRcvDataReady((tstime), (seqno), seqdistance)))
+        if (stillConnected() && !timeout && !(m_pRcvBuffer->isRcvDataReady((tstime), (seqno), seqdistance)))
```

Comment:
What does it fix actually? If the parenthesis isn't required for NOT, then it's not required anyway.

Reply:
I would delete them actually. I was trying to restore parentheses but misplaced one 🙂